lib/src/dartdoc_options.dart

```diff
@@ -727,10 +730,13 @@ abstract class DartdocOption<T> {
   /// Calls [valueAt] with the working directory at the start of the program.
   T valueAtCurrent() => valueAt(_directoryCurrent);
 
+  Folder __directoryCurrent;
   Folder get _directoryCurrent =>
-      resourceProvider.getFolder(resourceProvider.pathContext.current);
+      __directoryCurrent ??= resourceProvider.getFolder(_directoryCurrentPath);
```

Comment:
LOL I am pretty sure I've considered this optimization and dismissed it thinking I dunno what this crazy package does 😆 

Also, just to be sure; bizarro stuff like `@tool` shouldn't affect this, right?